sw/device/silicon_creator/lib/drivers/pinmux_unittest.cc

```diff
@@ -68,8 +68,12 @@ TEST_F(InitTest, WithBootstrap) {
   // The inputs that will be configured.
```

Comment:
@timothytrippel 
@alphan 
can we deprecate this unittest completely? it's somewhere being chip specific and a DIF.
Since there's an actual DIF, the unittest over there probably makes more sense..

Reply:
I think it would be best to keep this unit test. The mask ROM drivers are important and need CI testing too 😝

Reply:
o sorry, i thought this was under `sw/device/lib`